lib/npm-check-updates.js

```diff
@@ -433,6 +434,25 @@ async function run(options={}) {
     return await Promise.race([timeoutPromise, getAnalysis()]);
 }
 
+/**
+ * @param {PlainObject} [cfg]
+ * @param {string} [cfg.configFileName=".ncurc"]
+ * @param {string} [cfg.configFilePath]
+ * @param {string} [cfg.packageFile]
+ * @returns {PlainObject|undefined}
+ */
+function getNcurc({configFileName, configFilePath, packageFile} = {}) {
+    const rcFile = rcLoader('ncurc', {
+        configFileName: configFileName || '.ncurc',
+        defaultExtension: ['.json', '.yml', '.js'],
+        cwd: packageFile ?
+            path.dirname(packageFile) :
+            configFilePath || undefined
```

Comment:
`configFilePath` should have precedence over `packageFile` if present.

Reply:
Makes sense, thanks. Changed.